docs/marionette.behavior.md

```diff
@@ -316,10 +321,23 @@ including:
 * [`modelEvents`](./marionette.view.md#model-events)
 * [`collectionEvents`](./marionette.view.md#collection-events)
 
+### Events Order
+
+If both view and behavior are listening for the same event, this will be executed
+first in the view than in the behavior. 
+
+Unlike events, the initialize method will 
+be called first in the behavior. 
+  
+> While initialize is called in behavior before than in view, the view property is already set, and can be accessed, inside behavior initialize 
+  
+[Live example](https://jsfiddle.net/9wtqgwgu/)  
```

Comment:
@denar90 , thanks. #1579 was one of my source of information although the initial order is not valid anymore. I updated the fiddle to show constructor: https://jsfiddle.net/marionettejs/qb9go1y3/1/ .

This knowledge was one that i missed and had to test myself when i was learning how to use behavior. As the other changes i'm proposing, it is inspired by the lacks i observed reading the docs.